application/templates/index.html

```diff
@@ -6,7 +6,9 @@
 	<body>
 		<h1>Dashi</h1>
 		<h2>PRE-EXOME</h2>
-		<h3><a href="/preexome">PRE-EXOME Report</a></h3>
+		<h3><a href="/preqc-exome">PRE-EXOME Report</a></h3>
```

Comment:
did the URL for the pre-exome report change?

Reply:
^ Good point. @apmasell if you're making this change, the `page_name` will have to be changed in `pre_exome.py`

Reply:
I'm still very confused associations between URL, file name, `page_name` and that list.

Reply:
We should probably standardize them